### DISCUSSION TITLE
Issues running hello_world example

### DISCUSSION THREAD

**Discussion by etleyden** [CE-Programming/toolchain#415] [General]
In both CEmu and on my physical CE, when I build and run the hello_world DEMO.8xp, the program throws a syntax error. I have tried multiple versions of the toolkit, and am unsure of what my next steps should be. 
<img width="1244" alt="Screen Shot 2022-10-30 at 12 14 22" src="https://user-images.githubusercontent.com/113561319/198889344-e98a5331-7176-4fb0-b3b3-77a64bcd1f53.png">

**Comment by adriweb** [comment]
What's your OS version?

If before 5.3, you need to put `Asm(` (from the catalog) before prgmDEMO or whatever it's called. But updating to 5.4 would be a better idea, see below why. 

if 5.3 or 5.4, that part is not needed anymore

if 5.5 or later, TI removed ASM so you'll need a "jailbreak" first to restore that otherwise you'll get an ERR:INVALID. See [arTIfiCE](https://yvantt.github.io/arTIfiCE/) .

**Reply by etleyden** [reply]
Thank you for the update. Using `Asm(` worked for me. I saw that ASM was removed in later OS's so I was trying to find a place to download the latest before that removal. Do you know where I can go for that?

**Comment by adriweb** [comment]
TI-Planet has backups of all the OSes, so for instance for 5.4.0: 
- OS+Apps bundle: https://tiplanet.org/forum/archives_voir.php?id=2236017
- just the OS: https://tiplanet.org/forum/archives_voir.php?id=2236014
